packages/jsii-pacmak/test/generated-code/__snapshots__/examples.test.ts.snap

```diff
@@ -0,0 +1,1437 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Does this require another new snapshot test? 

Did this not manifest in the previous snapshot test for diamond dependencies - https://github.com/aws/jsii/pull/2591/files#diff-305aec203005012473bdfe3d0c5d9c734c7ee318a0e813759c82aed7067323e3 ?

Snapshot tests make it really hard to evaluate if a problem was actually fixed. It's useful to detect unintended regressions but perhaps it shouldn't be the sole testing mechanism.

Reply:
The alternative is to build (I mean compile) the generated code, for possibly many small projects, to ensure they "work" -- this is bound to be very slow/expensive. Also - next up is "but the fact the code compiles is not enough - we have to run it, too", and then it becomes a full time job?